tests/testthat/test-writeManifest.R

```diff
@@ -338,6 +338,69 @@ test_that("Sets environment.environment_management in the manifest if envManagem
   expect_null(manifest$environment$environment_management)
 })
 
+test_that("environment.r.requires - DESCRIPTION file - existing Depends R is added", {
+  skip_on_cran()
```

Comment:
You didn't start this, but looking at this file, every single `test_that` block starts with `skip_on_cran()` you can also (and honestly if we are skipping all of these should) just put `skip_on_cran()` at the top of the file and the full file will be skipped.